lib/github-changelog.php

```diff
@@ -452,48 +561,33 @@ function create_changelog_for_last_release() {
 
 	$prs = fetch_prs_by_ids( $pr_ids );
 
-	$all_changelog_html = '';
-	$all_tags           = array();
-
-	foreach ( $prs as $pr ) {
-		$changelog_html = get_changelog_html( $pr, false );
-
-		if ( ! empty( $changelog_html ) ) {
-			$all_changelog_html .= "\n\n" . $changelog_html;
-			$pr_tags             = get_changelog_tags( $pr['labels'] );
-			$all_tags            = array_merge( $all_tags, $pr_tags );
-		}
+	if ( empty( $prs ) ) {
+		echo "No PRs found for the latest release.\n";
+		exit( 0 );
 	}
 
-	if ( empty( $all_changelog_html ) ) {
+	list( $changelog_html, $changelog_tags ) = generate_changelog_from_prs( $prs );
+
+	if ( empty( $changelog_html ) ) {
 		echo "No changelog entries found in any PRs for this release.\n";
 		exit( 0 );
 	}
 
-	if ( count( $prs ) > 1 ) {
-		$all_changelog_html = aggregate_changelog_headings( $all_changelog_html );
```

Comment:
Curious here: could removing `aggregate_changelog_headings` lead to duplicated entries in the changelog or we're now solving this somewhere else that I'm missing?